README.md

```diff
@@ -13,6 +14,18 @@
 - [Troubleshooting](#troubleshooting)
   * [All Slaves are marked as offline](#all-slaves-are-marked-as-offline)
 
+
+## Create a persistent disk Jenkins home ##
+
+Change volumes partition from 1 to 0 in the file k8s/jenkins/jenkins.yaml\<br> 
```

Comment:
This is markdown, you don t need to add html tag. Please also add a script section (like above) with an example.